### PR TITLE
feat(connectors): add replacing deprecated template guide

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._
Write documentation on migrating to dynamic templates #1249

## Are there related marketing activities
https://github.com/camunda/camunda-platform-docs/issues/1249
_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._
https://github.com/camunda/team-connectors/issues/93
## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

ASAP would be ideal.

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by igpetrov** [comment]
@PreetiNirwal: "let's keep it generic" - note from daily

**Comment by christinaausley** [comment]
@Oleksiivanov Are you looking for my review here as well? If so, feel free to request it when you're ready 👍

File: docs/components/integration-framework/connectors/out-of-the-box-connectors/rest.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ description: Make a request to a REST API and use the response in the next steps
 
 The **REST Connector** allows you to make a request to a REST API and use the response in the next steps of your process.
 
+If you see that connector mark as 'Deprecated', you can update and replace it.
+See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more.
+
```
Comment (PreetiNirwal) [inline docs/components/integration-framework/connectors/out-of-the-box-connectors/rest.md:11]: @Oleksiivanov no need to add these sentence to REST Connector docs. 
You can remove entire: 

"If you see that connector mark as 'Deprecated', you can update and replace it.
See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more."

File: docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ description: Quickly send emails from your BPMN processes.
 
 The **SendGrid Connector** allows you to quickly send emails from your BPMN processes.
 
+If you see that connector mark as 'Deprecated', you can update and replace it.
+See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more.
+
```
Comment (PreetiNirwal) [inline docs/components/integration-framework/connectors/out-of-the-box-connectors/sendgrid.md:11]: @Oleksiivanov same as above, no need to add these sentence to SendGrid Connector docs.
You can remove entire:

"If you see that connector mark as 'Deprecated', you can update and replace it.
See [Replacing Templates](https://github.com/components/modeler/desktop-modeler/element-templates/using-templates.md) for more."

**Review by PreetiNirwal — CHANGES_REQUESTED** [review]
See comments. No need to update rest.md & sendgrid.md files.

**Review by PreetiNirwal — APPROVED** [review]
Looks good! Thanks.

**Comment by PreetiNirwal** [comment]
Hi @akeller, if we can merge this PR to the 8.0 docs when possible, that'd be great. Thanks.

**Comment by akeller** [comment]
Looks like this got merged without a review from the docs side. @christinaausley can you review this an open a separate PR with any adjustments?

**Comment by akeller** [comment]
> Hi @akeller, if we can merge this PR to the 8.0 docs when possible, that'd be great. Thanks.

@PreetiNirwal because these changes went into `/docs/` they will be available in `Next` (aka 8.1). Should this be available in the current docs (8.0)?

**Comment by PreetiNirwal** [comment]
Hi @akeller - this one we would like in 8.0. Hope that doesn't add complexity.

**Comment by akeller** [comment]
Nope, I'll take care of it 🙂

**Pull request merged**